apps/linear_algebra/Makefile

```diff
@@ -6,7 +6,7 @@ CXXFLAGS += -Wall -march=native -I /opt/local/include
 HL_TARGET ?= host
 LIBHALIDE_BLAS = $(BIN)/libhalide_blas.a
 HALIDEBLAS_FLAGS ?= -DUSE_HALIDE
-EMIT_OPTIONS = stmt,assembly,o,h
+EMIT_OPTIONS = stmt,assembly,object,h
```

Comment:
h -> c_header, based on what you have elsewhere

Reply:
done